lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
@jcoyne but it does not autoload and without this the Legato fails and gives an error " undefined method sufia__download". If I take out this line require 'sufia/analytics' only, even then I get an error. I have compared this with Sufia 6 and this is what was added in one of the PRs. Yesterday we looked at this with @mjgiarlo